src/main/java/org/jabref/gui/fieldeditors/URLUtil.java

```diff
@@ -62,7 +63,8 @@ public static String cleanGoogleSearchURL(String url) {
                 }
             }
             return url;
-        } catch (MalformedURLException e) {
+        } catch (
+                MalformedURLException e) {
```

Comment:
Remove empty line (we know that the IntelliJ autoformatter creates this - but no one invested significant time to fix this)